packages/user-mgnt/src/features/system/handler.ts

```diff
@@ -45,7 +57,11 @@ const pickSystem = (system: ISystemModel & { _id: Types.ObjectId }) => ({
   // TODO: client_id and sha_secret should be camelCased in the Mongoose-model
   _id: system._id.toString(),
   shaSecret: system.sha_secret,
-  clientId: system.client_id
+  clientId: system.client_id,
+  webhookPermissions: system.settings.webhook.map((ite) => ({
+    event: ite.event,
+    permissions: ite.permissions
+  }))
```

Comment:
IMO it would be ideal to have the System model in MongoDB and the System model user-mgnt returns as similar as possible to avoid confusion. Why do we need to transform the `system.settings.webhook` into a `system.webhookPermissions`?